### PR TITLE
Update chat link [ci-skip]

### PR DESCRIPTION
**Pull request by NotWoods**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Task Status](https://github.taskcluster.net/v1/repository/mozilla-mobile/android-components/master/badge.svg)](https://github.taskcluster.net/v1/repository/mozilla-mobile/android-components/master/latest)
 [![codecov](https://codecov.io/gh/mozilla-mobile/android-components/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla-mobile/android-components)
 [![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/19637)
-[![Gitter](https://badges.gitter.im/mozilla-mobile/android-components.svg)](https://gitter.im/mozilla-mobile/android-components)
+[![Gitter](https://img.shields.io/badge/chat-on%20matrix-51bb9c)](https://chat.mozilla.org/#/room/#android-components:mozilla.org)
```
Comment (kbrosnan) [inline README.md]: [![chat.mozilla.org] instead of Gitter?

Comment (NotWoods) [reply README.md]: Ack, fixed this and forgot to commit

**Comment by kbrosnan** [comment]
I don't have permission on android-components to do reviews. You will need to select someone else.

**Comment by NotWoods** [comment]
bors r=kbrosnan

**Pull request merged**